angularFiles.js

```diff
@@ -189,21 +189,78 @@ var angularFiles = {
     'src/angular.bind.js'
   ],
 
-  'karmaModules': [
+  'karmaModulesBase': [
     'build/angular.js',
-    '@angularSrcModules',
+    'src/ngMock/*.js',
```

Comment:
Does this have to be here? Can't it shadow problems as before (e.g. modules depending on stuff that is defined in `ngMock`'s source code)?

Reply:
the angular-mocks file defines the module fn that integrates with jasmine, so I think it unfortunately must always be included: https://github.com/angular/angular.js/blob/bb5a7e39ba2c8fe5afe1ec6f90ebedf22658a1c8/src/ngMock/angular-mocks.js#L3085

Reply:
But we get the same integration with `build/angular-mocks.js`, right? (Without the problem of making things defined in the `ngMocks` closure available to other modules.)

I would expect `build/angular-mocks.js` to be included in all module tests, except for `ngMocks` specs, which should indeed use `src/ngMock/*.js`. Do I miss something?

Reply:
Using `build/angular-mocks.js` works great, good suggestion!
In the beginning, it was a big hassle to separate all these files, it's possible that at some point the ngMock src files had to be included.